pkg/services/alerting/conditions/query.go

```diff
@@ -128,18 +136,28 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange *
 
 		result = append(result, v.Series...)
 
+		queryResultData := map[string]interface{}{}
+
 		if context.IsTestRun {
+			queryResultData["series"] = v.Series
+		}
+
+		if context.IsDebug && v.Meta != nil {
```

Comment:
Should probably only check if `v.Meta != nil` which will add query for sql datasources out of the box for example.

Reply:
No this should be correct. Only include if debug is true